src/test/components/ZipFileTree.test.js

```diff
@@ -63,19 +64,43 @@ describe('calltree/ZipFileTree', function () {
     [
       'foo',
       'bar',
-      'profile1.json',
-      'profile2.json',
+      'profile1',
+      'profile2',
       'baz',
-      'profile3.json',
+      'profile3',
+      'profile4',
     ].forEach((fileName) => screen.getByText(fileName));
   });
 
+  it('removes .json and .json.gz extensions', async () => {
+    await setup();
+
+    ['profile1', 'profile2', 'profile3', 'profile4'].forEach((fileName) =>
+      screen.getByText(fileName)
```

Comment:
We use the jest-dom library, which makes it possible to be somewhat more explicit in our expectations, with something such as:
```
expect(screen.getByText(..)).toBeInTheDocument()
```

Normally there's a eslint rule for that, but I think it doesn't find it because of the `forEach`

Reply:
ah yes I see
I believe the main reason is that this wasn't caught by the eslint rule :-)
Thanks for updating